clientcontroller/babylon_consumer.go

```diff
@@ -211,7 +212,7 @@ func (bc *BabylonConsumerController) SubmitBatchFinalitySigs(
 		return nil, err
 	}
 
-	return &types.TxResponse{TxHash: res.TxHash, Events: res.Events}, nil
+	return &types.TxResponse{TxHash: res.TxHash, Events: bc.fromCosmosEventsToBytes(res.Events)}, nil
```

Comment:
```suggestion
	return &types.TxResponse{TxHash: res.TxHash}, nil
```
This is not used in tests